trajectory_msgs/MultiDOFJointTrajectory trajectory
float32[] length_catenary